sticker/src/tensorflow/tagger.rs

```diff
@@ -238,32 +238,38 @@ where
             &self.graph.top_k_probs_op,
         )?;
 
-        // Convert label numbers to labels.
+        // Decode label numbers.
         let max_seq_len = tag_tensor.dims()[1] as usize;
         let k = tag_tensor.dims()[2] as usize;
-        let mut labels = Vec::new();
-        for (idx, sentence) in sentences.iter().enumerate() {
-            let seq_len = min(max_seq_len, sentence.borrow().len() - 1);
+        for (idx, sentence) in sentences.iter_mut().enumerate() {
+            // BorrowMut derives from borrow, but the borrowed type cannot
+            // be inferred with a borrow() call here?
+            let seq_len = min(
+                max_seq_len,
+                // Help the type checker, can't infer borrowed type otherwise.
+                <BorrowMut<Sentence> as Borrow<Sentence>>::borrow(sentence).len() - 1,
+            );
```

Comment:
```Rust
let sentence = sentence.borrow_mut();
let seq_len = min(
    max_seq_len,
    sentence.len() - 1,
);
```
would make this more readable, `.borrow_mut()` on line 269 could then be removed, too.

Reply:
Or map the iterator:
```Rust
for (idx, sentence) in sentences.iter_mut().map(BorrowMut::borrow_mut).enumerate() {
    //[...]
}
```